test/libraries/MultiSend.spec.ts

```diff
@@ -174,5 +176,58 @@ describe("MultiSend", async () => {
                 ),
             ).to.be.eq("0x" + "baddad".padEnd(64, "0"));
         });
+
+        it("can bubble up revert message on call", async () => {
+            const { delegatecaller, multiSend, mock } = await setupTests();
+            await mock.givenCalldataRevertWithMessage("0xbaddad", "Some random message");
+
+            const txs: MetaTransaction[] = [
+                {
+                    to: mock.address,
+                    value: 0,
+                    data: "0xbaddad",
+                    operation: 0,
+                },
+            ];
+            const { data } = buildMultiSendSafeTx(multiSend, txs, 0);
+
+            const { success, returndata } = await delegatecaller.callStatic.makeDelegatecal(multiSend.address, data);
+            expect(success).to.be.false;
+            expect(returndata).to.be.equal(
+                "0x08c379a000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000013536f6d652072616e646f6d206d65737361676500000000000000000000000000",
+            );
+        });
+
+        it("can bubble up revert message on delegatecall", async () => {
+            const { delegatecaller, multiSend, mock } = await setupTests();
+
+            const { data: setRevertMessageData } = await mock.populateTransaction.givenCalldataRevertWithMessage(
```

Comment:
`encodeFunctionData` seems more appropriate here, no?

Reply:
It does